### DISCUSSION TITLE
Laravel 9 upgrade

### DISCUSSION THREAD

**Discussion by pzubcak** [Jeroen-G/Explorer#85] [General]
Upgrade to laravel 9 is not possible. Composer.json in master branch looks good, but composer.json in latest release have only laravel 7.*, 8.* versions

composer update result:
 Root composer.json requires jeroen-g/explorer ^2.4 -> satisfiable by jeroen-g/explorer[v2.4.0, 2.4.1, v2.5.0, 2.5.1].
    - Conclusion: don't install laravel/framework v9.0.0-beta.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.4 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.5 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.1 (conflict analysis result)
    - jeroen-g/explorer[v2.4.0, ..., 2.5.1] require illuminate/support ~7|~8 -> satisfiable by illuminate/support[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev].
    - Only one of these can be installed: illuminate/support[v5.6.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev], laravel/framework[v9.0.0-beta.1, ..., 9.x-dev]. laravel/framework replaces illuminate/support and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^9.0 -> satisfiable by laravel/framework[v9.0.0-beta.1, ..., 9.x-dev].

**Comment by Jeroen-G** [comment]
Yes, I am planning to support Laravel 9 from Explorer 3 (because of a new Scout release and PHP 8). I have not yet tagged it, but it should already be stable. Like in the demo app you can thus require the `dev-master` version for the time being.
